src/main/java/com/gdschongik/gdsc/domain/studyv2/api/StudentStudyControllerV2.java

```diff
@@ -41,4 +43,18 @@ public ResponseEntity<StudentStudyMyCurrentResponse> getMyCurrentStudies() {
         var response = studentStudyServiceV2.getMyCurrentStudies();
         return ResponseEntity.ok(response);
     }
+
+    @Operation(summary = "수강중인 특정 스터디의 할 일 리스트 조회", description = "나의 수강중인 특정 스터디의 할 일 리스트를 조회합니다.")
+    @GetMapping("/{studyId}/me/todo")
```

Comment:
리스트 조회라는 관점에서 봤을 때 복수가 맞는 것 같아요!
```suggestion
    @GetMapping("/{studyId}/me/todos")
```